common/requirements.txt

```diff
@@ -1,4 +1,3 @@
--i https://artifactory.delphix.com/artifactory/api/pypi/delphix-virtual-pypi/simple/
 atomicwrites==1.3.0
 attrs==19.3.0
 configparser==4.0.2 ; python_version < '3'
```

Comment:
Just curious, any idea why we need these?

Reply:
I just copied the dev requirements.txt generated by pipenv with the old Gradle build. don't see where we user `atomicwrites` or `attrs`. We do use `configparser` but only in `tools`.

Reply:
Actually, I don't see where any of these are used. I just deleted them from all requirements.txt files and re-ran unit tests.